Casks/font/font-d/font-darwin.rb

```diff
@@ -0,0 +1,15 @@
+# typed: true
+# frozen_string_literal: true
+
+cask "font-darwin" do
```

Comment:
```suggestion
cask "font-darwin" do
```
There should be no comments in Cask declaration.